README.md

```diff
@@ -7,7 +7,9 @@ Plain simple build step for setting up your project with [Carthage](https://gith
 You can cache the result of `carthage bootstrap` to make Bitrise faster when building unchanged dependencies. For that you'll have to [set up caching](http://devcenter.bitrise.io/caching/about-caching/) in your Bitrise workflow.
 
 1. add a cache pull step before your Carthage step,
-2. set up a cache push step after your Carthage step and make sure to add `./Carthage -> ./Carthage/Cachefile`.
+2. set up a cache push step after your Carthage step and make sure to add:
+ * If you use Carthage v0.20.0 or later, `./Carthage`.
+ * Otherwise, `./Carthage -> ./Carthage/Cachefile`.
```

Comment:
We need to take Swift version update into consideration, so just `./Carthage` would be fine.
Here is a doc about Carthage's version file.
https://github.com/Carthage/Carthage/blob/master/Documentation/VersionFile.md